deployer/pipeline_deployer.py

```diff
@@ -106,6 +109,21 @@ def _check_experiment_name(self, experiment_name: Optional[str] = None) -> str:
 
         return experiment_name
 
+    def _check_pipeline_display_name(self, tag: Optional[str] = None) -> None:
```

Comment:
To utils ? WDYT ?

Reply:
As it it ressembles, the `_check_experiment_name` private method - line 103 - I thought it made sense to include it here too.
I can put it in utils if you think it does not.

Reply:
ok for me to keep it where it is